NGitLab/Impl/TagClient.cs

```diff
@@ -25,16 +25,17 @@ public void Delete(string name)
             _api.Delete().Stream($"{_tagsPath}/{WebUtility.UrlEncode(name)}", _ => { });
         }
 
-        public RealeaseInfo CreateRelease(string name, ReleaseCreate data)
+        public ReleaseInfo CreateRelease(string name, ReleaseCreate data)
```

Comment:
Theses methods should be deprecated too.

Reply:
After checking our code, we don't use those routes so I'm fine if we just remove them. That way we don't need to duplicate `ReleaseCreate` / `ReleaseUpdate`.

Reply:
Like you did at the beginning... 😅 (but keep `ReleaseDescription` in `TagCreate`)

Reply:
I've removed them 😄